packages/tictactoe/config/paths.js

```diff
@@ -5,20 +5,20 @@ const fs = require('fs');
 const url = require('url');
```

Comment:
At some point soon, we should probably factor out configuration for rps and ttt into a separate package.